backend/src/core/loaders/express.loader.ts

```diff
@@ -71,7 +71,13 @@ const expressApp = ({ app }: { app: express.Application }): void => {
   // in the parseEvent() handle before parsing the SES event.
   app.use('/v1/callback/email', express.text({ type: 'application/json' }))
 
-  app.use(express.json())
+  app.use(
+    express.json({
+      // this must be significantly bigger than transactionalEmail.bodySizeLimit so that users who exceed limit
+      // will get 404 error informing them of the size of the limit, instead of 500 error
```

Comment:
Why are we returning 404, that's a bit weird

Reply:
If it's because we're leaving it to the default, what could probably work is to update the [error handler](https://github.com/opengovsg/postmangovsg/blob/aa7cf4f580b0e557ccd3f0c6ba9a27064eba769e/backend/src/core/loaders/express.loader.ts#L128-L141) following [this](https://stackoverflow.com/questions/58134287/catch-error-for-bad-json-format-thrown-by-express-json-middleware) to handle the error

Reply:
This should return 400 with a message informing user about the size limit

Reply:
oh wait I think the `404` might be a typo, I think it's really 400.

let me verify later ps

Reply:
Yup it's 400, sorry typo